### ISSUE TITLE
Mobile menu not scrolling on IPhone

### ISSUE THREAD

**Issue by UraraReika** [Crocoblock/suggestions#1528]
I'm using the jet menu for mega menus, but on mobile it's not scrolling correctly. Sometime is scrolls most of the time it doesn't.

1. Create menu
2. Add mega menu widget
3. Add some areas
4. open on Iphone

 - OS:  iOS
 - Browser  safari

 - Device:  iPhone

**Comment by pixallusdev** [comment]
This issue here is when Astra theme is activated, the mobile mega menu does not scroll properly. You have to pull the page down, let it settle, then is starts working properly. I've been waiting two months on this. please advise.

**Comment by MPHood** [comment]
How is this not fixed yet?
Many people are using IOS devices ( iphones)

Please guys fix this issue.

Ive got a client that expects the mega menu to work on IOS as well as it does on android.

**Comment by MPHood** [comment]
> This issue here is when Astra theme is activated, the mobile mega menu does not scroll properly. You have to pull the page down, let it settle, then is starts working properly. I've been waiting two months on this. please advise.

Its not an astra specific problem

**Comment by MPHood** [comment]
> This issue here is when Astra theme is activated, the mobile mega menu does not scroll properly. You have to pull the page down, let it settle, then is starts working properly. I've been waiting two months on this. please advise.

Its not related to Astra. I have the same issue and im using another theme

**Comment by robfaulkner75** [comment]
Hi, did anyone find a fix for this?  I am having this issue

**Comment by Alphecca38** [comment]
Yep, same here, it's been driving me crazy. Also on iphone with safari browser.

**Comment by robfaulkner75** [comment]
Hi @Alphecca38 are you using Siteground?  I found it was siteground optimiser that had the conflict.  I disabled and moved to WP Rocket and no more issues, it works really well now.  Hope this helps - it took me days to work this out :-)

**Comment by Alphecca38** [comment]
@robfaulkner75 Hi, no, I'm currently building the site (woocommerce store) on websupport.sk hosting (a good local hosting company in Slovakia), and I don't use any caching whatsoever (precisely to avoid such problems while building).

What does the siteground optimiser do? Maybe my hosting provider has a similar feature I'm not aware of.

**Comment by robfaulkner75** [comment]
@Alphecca38 it optimises performance, speed, caching, image optimisation etc.  The issue was a plugin / cache conflict with jet menu and SG Optimizer.  I cleared my cache on SGround and then locally (after installing WP Rocket).  WP Rocket enabled me to have separate cache files for desktop and mobile.  This fixed my issue - although it did take 24 hrs after cleaning the cache to resolve.   Hope this helps you

**Comment by Alphecca38** [comment]
@robfaulkner75 Thanks for sharing! I'll look into it, maybe there's something similar causing the issue in my case.
